### PR TITLE
updated tests for item/view, item/module

### PR DESCRIPTION
**Pull request by anastasiyaSht**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=777&view=detail&selectedIssue=LEAR-87)

## Code reviewers

- [ ] @Sokolovskyi1995
- [x] @BDubas
- [ ] @rromanitc
- [ ] @martasakal
- [ ] @Zaplatynskyi
- [ ] @aberezh

### Second Level Review

- [ ] @lhalam
- [ ] @ssOleg

## Summary of change
add tests: for item/testview : test_delete_not_success, test_delete_success, test_delete_if_user_not_mentor, test_delete_if_not_item, test_delete_if_not_topic, test_put_if_not_data, test_put_is_user_not_a_mentor, test_put_if_not_topic, test_put_if_not_item...

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/tests/unittests/item/testmodel.py [hunk 1]
```diff
@@ -195,16 +195,24 @@ def test_item_update(self):
 
         estimation = datetime.timedelta(seconds=104000)
         actual_item = Item.objects.get(id=101)
+        #
```
Comment (lhalam) [inline eventually/tests/unittests/item/testmodel.py]: remove `#`

File: eventually/tests/unittests/item/testmodel.py [hunk 2]
```diff
@@ -195,16 +195,24 @@ def test_item_update(self):
...
+        superior_first = Item.objects.get(id=102)
+        superior_second = Item.objects.get(id=103)
+        superiors = [superior_first, superior_second]
+        #
```
Comment (lhalam) [inline eventually/tests/unittests/item/testmodel.py]: remove `#`

File: eventually/tests/unittests/item/testmodel.py [hunk 3]
```diff
@@ -195,16 +195,24 @@ def test_item_update(self):
...
-                           estimation=estimation)
+                           estimation=estimation,
+                           superiors=superiors)
         expected_item = Item.objects.get(id=101)
         self.assertEqual(actual_item, expected_item)
         self.assertEqual(actual_item.name, 'listen book')
         self.assertEqual(actual_item.form, 2)
         self.assertEqual(actual_item.description, 'updated description')
         self.assertEqual(actual_item.estimation, estimation)
+        #
```
Comment (lhalam) [inline eventually/tests/unittests/item/testmodel.py:212]: remove `#`

File: eventually/tests/unittests/item/testmodel.py [hunk 4]
```diff
@@ -279,3 +287,6 @@ def test_item_none_delete(self):
 
         is_item_delete = Item.delete_by_id(188)
         self.assertIsNone(is_item_delete)
+
+    def test_get_item_superiors(self):
+        pass
```
Comment (lhalam) [inline eventually/tests/unittests/item/testmodel.py]: add endline

File: eventually/tests/unittests/item/testview.py [hunk 5]
```diff
@@ -79,6 +79,9 @@ def setUp(self):
         self.client = Client()
         self.client.login(username='email1@mail.com', password='1111')
 
+        self.client_second = Client()
```
Comment (lhalam) [inline eventually/tests/unittests/item/testview.py]: why do you need a second client?

**Review by lhalam — CHANGES_REQUESTED** [review]

File: eventually/tests/unittests/item/testview.py (hunk 5 above)
Comment (anastasiyaSht) [reply eventually/tests/unittests/item/testview.py]: I need second customer to check if user without permissions can delete topic

**Review by rromanitc — DISMISSED** [review]

**Review by BDubas — DISMISSED** [review]

**Review by Zaplatynskyi — DISMISSED** [review]

**Review by BDubas — DISMISSED** [review]

**Review by lhalam — DISMISSED** [review]

**Review by BDubas — APPROVED** [review]

**Review by martasakal — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**